Makefile

```diff
@@ -14,9 +14,9 @@ DEPLOY_BUILD_NUMBER ?= ${BUILD_NUMBER}
 DOCKER_CONTAINER_PREFIX = ${USER}-${BUILD_TAG}
 
 NOTIFY_CREDENTIALS ?= ~/.notify-credentials
-CF_MANIFEST_FILE ?= manifest-${CF_SPACE}.yml
 
 CF_APP ?= notify-antivirus
+CF_MANIFEST_FILE ?= manifest$(subst notify-antivirus,,${CF_APP}).yml.j2
```

Comment:
Is there a `-` missing here between `manifest` and `$(`?

Reply:
The app names are `notify-antivirus` and `notify-antivirus-api`, so the dash for `manifest-api` comes from the CF_APP itself

Reply:
Oh I see. 👍